components/ui/card.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from "react"
```

Comment:
Can't you simply do `import React from "react"`?

Reply:
This is also from shadcn